flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/PostgresDialect.java

```diff
@@ -86,7 +87,11 @@ public String getLimitClause(long limit) {
 
     @Override
     public String quoteIdentifier(String identifier) {
-        return identifier;
+        if (identifier.contains(".")) {
```

Comment:
columns also could contain capital letters, probably should have a test with such column

Reply:
This test case  in PostgresCatalogTestBase#getUpperTable